Patch from Debian:
https://sources.debian.net/data/main/libe/libewf/20140608-6/debian/patches/04-fix-FTBFS-GCC5.patch

Description: fix a FTBFS with GCC-5. Thanks to Linn Crosetto <linn@hp.com> for
             the first fix (see #777938). This patch closes #777945.
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2015-07-02
Index: libewf-20140608/libuna/Makefile.am
===================================================================
--- libewf-20140608.orig/libuna/Makefile.am
+++ libewf-20140608/libuna/Makefile.am
@@ -3,7 +3,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/include \
 	-I$(top_srcdir)/common \
 	@LIBCSTRING_CPPFLAGS@ \
-	@LIBCERROR_CPPFLAGS@
+	@LIBCERROR_CPPFLAGS@ -std=gnu89
 
 noinst_LTLIBRARIES = libuna.la
 
